android/java/v1/AmpliApp/app/src/test/java/com/example/ampliapp/AmpliTest.java

```diff
@@ -92,92 +91,6 @@ public void identify() {
         );
     }
 
-    @Test
```

Comment:
Even though this is not part of the Ampli SDK, it could be good to keep the group tests, so that we know we can support the functionality via `ampli.client.<group-method>()`

Reply:
Restored.